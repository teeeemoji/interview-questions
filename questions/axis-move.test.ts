import axisMove from './axis-move'

test('A10;S20;W10;D30;X;A1A;B10A11;;A10;', () => {
  expect(axisMove('A10;S20;W10;D30;X;A1A;B10A11;;A10;')).toMatchSnapshot()
})
