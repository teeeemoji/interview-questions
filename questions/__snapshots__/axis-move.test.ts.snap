// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`A10;S20;W10;D30;X;A1A;B10A11;;A10; 1`] = `
Array [
  10,
  -10,
]
`;
